src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Col,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import CreatePasswordModal from './Modals/CreatePasswordModal';
+import ResetPasswordModal from './Modals/ResetPasswordModal';
+
+import css from './CreateResetPasswordControl.css';
+
+class CreateResetPasswordControl extends React.Component {
+  static propTypes = {
+    email: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    userId: PropTypes.string.isRequired,
+    resources: PropTypes.shape({
+      isLocalPasswordSet: PropTypes.shape({
+        records: PropTypes.object,
+      }).isRequired,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      resetPassword: PropTypes.shape({
+        POST: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+  };
+
+  static manifest = Object.freeze({
+    resetPassword: {
+      type: 'okapi',
+      path: 'bl-users/password-reset/link',
+      fetch: false,
+      throwErrors: false,
+    },
+    isLocalPasswordSet: {
+      type: 'okapi',
+      path: 'authn/credentials-existence?userId=!{userId}',
+    },
+  });
+
+  state = {
+    showModal: false,
+    link: '',
```

Comment:
Should we be more concise in naming here? Please, consider the possibility to name it as `createResetPasswordLink`
`showModal` is a boolean, wouldn't it be better to name it as `isModalOpen`?

Reply:
No, all names are clear